src/OrleansProviders/SQLServer/CreateOrleansTables_SqlServer.sql

```diff
@@ -105,11 +105,11 @@ GO
 -- These can be redefined (e.g. to provide non-destructive updates) provided the stated interface principles hold.
 CREATE TABLE [OrleansQuery]
 (	
-    [Key] VARCHAR(64) NOT NULL,
```

Comment:
This was originally an unfortunate choice as it is a reserved word in SQL Server and likely in other databases too.